lib/Yield/CrossRelations.cpp

```diff
@@ -83,11 +83,8 @@ static void conversionHelper(const yield::CrossRelations &Input,
     for (const auto &[RelationKind, TargetString] : Related) {
       switch (RelationKind) {
       case yield::RelationType::IsCalledFrom:
-        AddEdge(LocationString, TargetString, RelationKind);
-        break;
-
-      // TODO: handle dynamic function calls properly
       case yield::RelationType::IsDynamicallyCalledFrom:
+        AddEdge(LocationString, TargetString, RelationKind);
         break;
```

Comment:
* No fixup commits. "TODO: handle dynamic function calls properly" was introduced in an earlier commit.
* Do we still have "handle dynamic function calls properly" or are they all handled?

> location-dependent `toCallGraph`

The commit message is a bit confusing: `s/location-dependent/location-based/`.

> some minor call graph specific improvements

Vague

Reply:
Took responsibility for this, as some git shenanigans were required, and wanted to the relieve the poor @drblallo from this. So, the "TODO: handle dynamic function calls properly" fixup was fixed by squashing that change into the commit that introduced it (Ivan's one), plus some further fix down in the commits (to reflect this). The commit message has also been improved.